apps/frontend/src/libs/components/sidebar/utils/sidebar-items.ts

```diff
@@ -0,0 +1,17 @@
+import wheelOfBalance from "~/assets/img/wheel-of-balance.svg";
+import wheelOfBalanceInactive from "~/assets/img/wheel-of-balance-inactive.svg";
```

Comment:
I'd probably reverse the names of the files to `wheel-of-balance-active` (note the **active** part) and `wheel-of-balance` for the inactive one - so that the second one will have a _default_ icon, which is inactive in our case. What do you think on that?